### DISCUSSION TITLE
About console picture resolution

### DISCUSSION THREAD

**Discussion by changlin-ponyai** [ipxe/ipxe#1006] [Q&A]
How to increase the resolution of the console background image to 1920x1080 ？
When I set the background image to 1024x768, it gets stretched on different monitors.

**Comment by NiKiZe** [comment]
The documentation at https://ipxe.org/cmd/console should cover most scenarios.
If your picture is the same size as the console it will not be stretched.

But the video output, or the monitor might stretch the screen. This is outside of the control of iPXE.
This is also an "issue" in for example winpe, where the resolution often is 800x600 or similar, which is then stretched to 16:9 on the monitor.
